src/main/java/com/example/backendspringboottechiteasycontroller/controllers/TelevisionController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.backendspringboottechiteasycontroller.controllers;
+
+import com.example.backendspringboottechiteasycontroller.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    List<Television> televisionDatabase = new ArrayList();
+
+//    @GetMapping("/filter")
+//    public ResponseEntity<Object> getTelevisions(@RequestParam String brand, @RequestParam Integer maxPrice){
+//
+//        return ResponseEntity.ok(/*returnList*/);
+//    }
+    @GetMapping("/televisions")
+    public ResponseEntity<String> getTelevisions()
+    {
+        return ResponseEntity.ok("televisionDatabase");
+    }
+    @GetMapping("/television/{id}")
+    public ResponseEntity<Television> getSpecificTelevision(@PathVariable Integer id)
+    {
+        Television television = televisionDatabase.get(id);
+        if (television != null) {
+            return ResponseEntity.ok(television);
+        } else {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
+        }
+    }
+    @PostMapping("/television")
+    public ResponseEntity<Void> addTelevision(@RequestBody Television addTelevision)
+    {
+            televisionDatabase.add(addTelevision);
+            return ResponseEntity.created(null).build();
+    }
+    @PutMapping("/television/{id}")
+    public ResponseEntity<String> updateTelevision(@PathVariable Integer id)
```

Comment:
For a PUT mapping, you would need a pathvariable and a reuestbody. The pathvariable is to determine which entity you want to change (at which index in the list) and the requestbody is to determine what you want to change it into.

Reply:
You haven't implemented this yet. This method also needs to use the repository and return a `ResponseEntity<Television>`